examples/8.search_relevance/search_relevance.ipynb

```diff
@@ -2,109 +2,129 @@
  "cells": [
```

Comment:
<p>Let's either remove this output or keep other similar outputs</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Toloka/toloka-kit/pull/83/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>